both/api/licenses/server/cache-purging.js

```diff
@@ -0,0 +1,16 @@
+import { Meteor } from 'meteor/meteor';
+import { Licenses } from '../licenses';
+import addKeysToFastlyPurgingQueue from '../../../../server/cdn-purging/addKeysToFastlyPurgingQueue';
+
+// This automatically purges place JSON responses of this license, as all place responses
+// have a license included
+Meteor.startup(() => {
+  Licenses.find().observeChanges({
+    changed(_id) {
+      addKeysToFastlyPurgingQueue([_id]);
```

Comment:
0: It might make sense to prefix the ids with the type - there is no uniqueness constraint across collections. But yes - very unlikely.

Reply:
Hehe :D I was actually thinking about leaving out every second letter to not approach fastly's 16k surrogate key limit too fast for bigger responses…